pathd/path_pcep_debug.c

```diff
@@ -1320,7 +1322,7 @@ void _format_pcep_event(int ps, pcep_event *event)
 		PATHD_FORMAT("%*sevent_type: %s\n", ps2, "",
 			     pcep_event_type_name(event->event_type));
 		PATHD_FORMAT("%*sevent_time: %s", ps2, "",
-			     ctime(&event->event_time));
+			     ctime_r(&event->event_time, buf));
```

Comment:
any particular reason we did not fix tests/bgpd/test_mpath.c too?

Reply:
no, my bad - just didn't look into that one, I'll add it in...